### DISCUSSION TITLE
Translation / Internationalization

### DISCUSSION THREAD

**Discussion by Talanor** [projectdiscovery/nuclei#5163] [Ideas]
Hey.
I'd be willing to tranlate names/description/remediations in french for existing templates.
But I haven't seen a built in way as of right now to switch between languages or equivalent.
I could just fork/duplicate the templates and translate them but I'd much rather contribute than keep an entirely duplicated base.
Bedt

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
